.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
It looks like you can use `git clean -xe ...` to clean ignored files, but then specify a specific pattern to definitely keep. https://git-scm.com/docs/git-clean